### PR TITLE
Fix mongodb seeding error

### LINKED ISSUE #5: [ISSUE] mongo error on start project
Instead I use relational db, I choose mongo 
and then when it was importing data it shows this error
It works! I created all project and backend was working fine even with the fallow error but frontend not:
```
[develop:backend ] model {
[develop:backend ]   title: 'A bug is becoming a meme on the internet',
[develop:backend ]   content: "It's the story of a user named **Omer Barnir** who reported a bug in 2005 on the MySQL [bug report platform](https://bugs.mysql.com/)\n" +
[develop:backend ]     '\n' +
[develop:backend ]     "But the thing is that Omer never get an answer. 15 years later, the bug has never been fix and people are starting to make fun out of it. We let you take a look at the conversation [here](https://bugs.mysql.com/bug.php?id=11472), it's pretty funny",
[develop:backend ]   publishedAt: '2020-07-16T10:00:00.000Z',
[develop:backend ]   status: 'published',
[develop:backend ]   slug: 'a-bug-is-becoming-a-meme-on-the-internet',
[develop:backend ]   category: { id: 2 },
[develop:backend ]   author: { id: 2 },
[develop:backend ]   description: 'How a bug on MySQL is becoming a meme on the internet',
[develop:backend ]   image: null
[develop:backend ] } CastError: Cast to ObjectId failed for value "{ id: 2 }" at path "category"
[develop:backend ]     at model.Query.exec (/home/ai/Documentos/Projetos/meu-blog-strapi/backend/node_modules/mongoose/lib/query.js:4371:21)
[develop:backend ]     at model.Query.Query.then (/home/ai/Documentos/Projetos/meu-blog-strapi/backend/node_modules/mongoose/lib/query.js:4463:15)
[develop:backend ]     at processTicksAndRejections (internal/process/task_queues.js:97:5)
[develop:backend ]     at runNextTicks (internal/process/task_queues.js:66:3)
[develop:backend ]     at processImmediate (internal/timers.js:434:9) {
[develop:backend ]   messageFormat: undefined,
[develop:backend ]   stringValue: '"{ id: 2 }"',
[develop:backend ]   kind: 'ObjectId',
[develop:backend ]   value: { id: 2 },
[develop:backend ]   path: 'category',
[develop:backend ]   reason: Error: Argument passed in must be a single String of 12 bytes or a string of 24 hex characters
[develop:backend ]       at new ObjectID (/home/ai/Documentos/Projetos/meu-blog-strapi/backend/node_modules/bson/lib/bson/objectid.js:59:11)
[develop:backend ]       at castObjectId (/home/ai/Documentos/Projetos/meu-blog-strapi/backend/node_modules/mongoose/lib/cast/objectid.js:25:12)
[develop:backend ]       at ObjectId.cast (/home/ai/Documentos/Projetos/meu-blog-strapi/backend/node_modules/mongoose/lib/schema/objectid.js:267:12)
[develop:backend ]       at ObjectId.SchemaType.applySetters (/home/ai/Documentos/Projetos/meu-blog-strapi/backend/node_modules/mongoose/lib/schematype.js:1075:12)
[develop:backend ]       at ObjectId.SchemaType._castForQuery (/home/ai/Documentos/Projetos/meu-blog-strapi/backend/node_modules/mongoose/lib/schematype.js:1510:15)
[develop:backend ]       at ObjectId.SchemaType.castForQuery (/home/ai/Documentos/Projetos/meu-blog-strapi/backend/node_modules/mongoose/lib/schematype.js:1500:15)
[develop:backend ]       at ObjectId.SchemaType.castForQueryWrapper (/home/ai/Documentos/Projetos/meu-blog-strapi/backend/node_modules/mongoose/lib/schematype.js:1477:20)
[develop:backend ]       at castUpdateVal (/home/ai/Documentos/Projetos/meu-blog-strapi/backend/node_modules/mongoose/lib/helpers/query/castUpdate.js:528:19)
[develop:backend ]       at walkUpdatePath (/home/ai/Documentos/Projetos/meu-blog-strapi/backend/node_modules/mongoose/lib/helpers/query/castUpdate.js:266:22)
[develop:backend ]       at castUpdate (/home/ai/Documentos/Projetos/meu-blog-strapi/backend/node_modules/mongoose/lib/helpers/query/castUpdate.js:94:7)
[develop:backend ]       at model.Query._castUpdate (/home/ai/Documentos/Projetos/meu-blog-strapi/backend/node_modules/mongoose/lib/query.js:4556:10)
[develop:backend ]       at castDoc (/home/ai/Documentos/Projetos/meu-blog-strapi/backend/node_modules/mongoose/lib/query.js:4585:18)
[develop:backend ]       at model.Query._updateThunk (/home/ai/Documentos/Projetos/meu-blog-strapi/backend/node_modules/mongoose/lib/query.js:3754:20)
[develop:backend ]       at model.Query.<anonymous> (/home/ai/Documentos/Projetos/meu-blog-strapi/backend/node_modules/mongoose/lib/query.js:3852:23)
[develop:backend ]       at model.Query._wrappedThunk [as _updateOne] (/home/ai/Documentos/Projetos/meu-blog-strapi/backend/node_modules/mongoose/lib/helpers/query/wrapThunk.js:16:8)
[develop:backend ]       at /home/ai/Documentos/Projetos/meu-blog-strapi/backend/node_modules/kareem/index.js:278:20
[develop:backend ] }
```

To solve it I just updated imported articles selecting image cartegory and writer
and I found a bug on frontend when the article has no image 
to solve this I just did on my-blog/frontend/components/card.js
```
<Image image={article.image || {}} />
```
the full component:
```


import React from "react";
import Link from "next/link";
import Image from "./image";

const Card = ({ article }) => {
  return (
    <Link as={`/article/${article.slug}`} href="/article/[id]">
      <a className="uk-link-reset">
        <div className="uk-card uk-card-muted">
          <div className="uk-card-media-top">
            <Image image={article.image || {}} />
          </div>
          <div className="uk-card-body">
            <p id="category" className="uk-text-uppercase">
              {(article.category || {}).name}
            </p>
            <p id="title" className="uk-text-large">
              {article.title}
            </p>
          </div>
        </div>
      </a>
    </Link>
  );
};

export default Card;

```
After I solve this I update Alternative text and Caption on each image
on media Library

them all works fine 

I hope it can help someone.. 

and I suggest a fix to instead of try to set Id it gets the id from mongo and update the collections 
because the problem was because it was trying to add a relational id to a non relation db

### PR DESCRIPTION
**Pull request by markkaylor**
This is maybe not the best solution but a workaround that avoids an error when using mongodb.

- Relations are set randomly in the bootstrap function using `_id` for mongodb and `id` for sql databases

closes #5 , closes #7

### CODE REVIEW

**Review by remidej — CHANGES_REQUESTED** [review]
You didn't lie, that approach is quite creative 😄 

I think having random relations is much better than an error. But it's also not as good as having the right ones all the time. So do you think we could only keep the "random" solution for Mongoose, and keep the current system for SQL databases?

Also, it looks like you've applied some code formatting here. But I think the previous style matches the Strapi codebase better, since all Strapi files use double quotes and semi colons

File: template/config/functions/bootstrap.js [hunk 1]
```diff
@@ -108,30 +110,69 @@ async function importHomepage() {
...
-    return createEntry({ model: "article", entry: article, files });
-  }));
+  const categories = await strapi.query("category").find();
+  const authors = await strapi.query("writer").find();
+
+  return Promise.all(
+    articles.map((article) => {
+      // Get relations for each article
+      const entry = getEntryWithRelations(article, categories, authors);
+      console.log(entry);
```
Comment (remidej) [inline template/config/functions/bootstrap.js]: Let's remove this log

**Review by remidej — COMMENTED** [review]
It's working well 🙂 

Just remove the remaining console log then we can merge this

File: template/config/functions/bootstrap.js (hunk 1 above)
Comment (markkaylor) [reply template/config/functions/bootstrap.js]: woops, will do

**Review by remidej — APPROVED** [review]

**Pull request merged**